tests/tests_unit/test_api/test_assets.py

```diff
@@ -122,6 +122,16 @@ def test_list_root(self, mock_assets_response):
             calls[0].request.body
         )
 
+    def test_list_subtree(self, mock_assets_response):
+        ASSETS_API.list(asset_subtree_ids=1, asset_subtree_external_ids=["a"], limit=10)
```

Comment:
We should use these subtree filter params in Asset.subtree() and AssetsApi.retrieve_subtree().